src/pages/dashboard/ui/mail/EmailPage.tsx

```
@@ -4,25 +4,55 @@ import TimePicker from '../../../../features/event-manage/event-create/ui/TimePi
+    };
+
+    sendEmail(emailData);
+  };
 
   return (
     <DashboardLayout centerContent="WOOACON 2024">
       <div className="p-5 flex flex-col gap-10 min-h-full">
         <EmailInput
           openSelectTicket={() => setTicketModalOpen(true)}
-          allParticipantEmails={participants.map(p => p.email)}
+          allParticipantEmails={participants.map((p: { email: any; }) => p.email)}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)